### PR TITLE
[Side nav alt] Creating side nav alt with chevrons.

### PR DESCRIPTION
**Pull request by adrianconte**

### CODE REVIEW

**Comment by adrianconte** [comment]
Adds 'side nav alternative' (currently used on health.gov.au.)
image

File: docs/components/detail/abbreviations.html [hunk 1]
```diff
@@ -3256,6 +3267,35 @@ <h4 class="Tree-collectionLabel" data-role="toggle">
         
     
         
+        
+        <li class="Tree-item Tree-entity" data-role="item">
+            <a class="Tree-entityLink" href="side-nav--alternative.html" data-pjax>
+                <span>Alternative</span>
+                
+
+<div class="Status Status--unlabelled">
```
Comment (Benjen) [inline docs/components/detail/abbreviations.html:3276]: @adrianconte could you look at the formatting of this section and update accordingly in each of the relevant files.

File: docs/components/detail/abbreviations.html [hunk 2]
```diff
@@ -3232,10 +3232,21 @@ <h4 class="Tree-collectionLabel" data-role="toggle">
         
     
         
+        <li class="Tree-item Tree-collection Tree-depth-3" data-behaviour="collection" id="tree-components-collection-side-nav">
+            <h4 class="Tree-collectionLabel" data-role="toggle">
+                <span>Side Nav</span>
+            </h4>
+            <ul class="Tree-collectionItems" data-role="items">
+
```
Comment (Benjen) [inline docs/components/detail/abbreviations.html:3240]: @adrianconte there is a lot of vertical spacing in here. Could you please check the formatting and update accordingly in each relevant file.

**Review by Benjen — CHANGES_REQUESTED** [review]
There is a lot of duplication of markup here. I am sure there is a way we can reduce using some kind of extension/inclusion functionality in Handlebars. Actually I know there is a lot of similar duplication in many of the HDS components. We had planned to address this in HDS v2.x, but unfortunately that never got off the ground due to lack of resources :-(

Any idea of how difficult it would be to refactor these particular changes to make it a bit more DRY?

**Pull request merged**